cmd/init-repo/main.go

```diff
@@ -17,8 +17,9 @@ import (
 )
 
 const (
-	depCommit   = "7c44971bbb9f0ed87db40b601f2d9fe4dffb750d"
-	godepCommit = "tags/v80"
+	depCommit      = "7c44971bbb9f0ed87db40b601f2d9fe4dffb750d"
+	godepVersion   = "v80-k8s-r1"
+	k8sGodepCommit = "d5f2096f1a37a31b9f450d601411b8a85b64c624"
```

Comment:
This is the latest commit on the `release-1.14` branch - https://github.com/kubernetes/kubernetes/commit/d5f2096f1a37a31b9f450d601411b8a85b64c624